app/main.py

```diff
@@ -1,3 +1,17 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return int(days / 7) + (days % 7 > 0)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        return cls(
+            course_dict["name"],
+            course_dict["description"],
+            cls.days_to_weeks(course_dict["days"])
```

Comment:
It would be better to use `OnlineCourse.days_to_weeks` here. You will discuss this case in the future videos on OOP